test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -288,7 +316,17 @@ void ExpressionEvaluator::operator()(FunctionCall const& _funCall)
 		if (BuiltinFunctionForEVM const* fun = dialect->builtin(_funCall.functionName.name))
 		{
 			EVMInstructionInterpreter interpreter(m_state, m_disableMemoryTrace);
-			setValue(interpreter.evalBuiltin(*fun, _funCall.arguments, values()));
+
+			u256 const value = interpreter.evalBuiltin(*fun, _funCall.arguments, values());
```

Comment:
If `fun` is an external function here, isn't it then already evaluated? E.g. didn't it ran already?

Reply:
It couldn't be external, this branch is only entered for builtin functions...

Reply:
Ah yes! True!